app/javascript/shared/components/useKeyboardShortcuts.js

```diff
@@ -12,22 +12,51 @@ function isFormField(element) {
   if (element instanceof HTMLElement === false) return false;
 
   const name = element.nodeName.toLowerCase();
-  const type = (element.getAttribute('type') || '').toLowerCase();
+  const type = (element.getAttribute("type") || "").toLowerCase();
   return (
-    name === 'select' ||
-    name === 'textarea' ||
-    (name === 'input' &&
-      type !== 'submit' &&
-      type !== 'reset' &&
-      type !== 'checkbox' &&
-      type !== 'radio') ||
+    name === "select" ||
+    name === "textarea" ||
+    (name === "input" &&
+      type !== "submit" &&
+      type !== "reset" &&
+      type !== "checkbox" &&
+      type !== "radio") ||
     element.isContentEditable
   );
 }
 
+/**
+ * Function to handle 
```

Comment:
refactor (blocking) - I love that you've added JSDoc here! It would be great though if you could add some more detail about what is being handled here

Reply:
I clearly started writing it and get distracted 😅